eventlog/cloudwatch/cloudwatch.go

```diff
@@ -212,7 +212,7 @@ func (c *CloudWatchEventer) loop(ctx context.Context) {
 		})
 		if err != nil {
 			log.Error.Printf("CloudWatchEventer: PutLogEvents error: %v", err)
-			if aerr, ok := err.(cloudwatchlogs.InvalidSequenceTokenException); ok {
+			if aerr, ok := err.(*cloudwatchlogs.InvalidSequenceTokenException); ok {
```

Comment:
Is this something that changed with the update or just some unrelated thing that you noticed was broken?

Reply:
This appeared to be required to make things compile after upgrading `github.com/aws/aws-sdk-go`.

Reply:
Ah, perhaps because the non-pointer type can no longer be an `error` and the type assertion can be determined to always fail.